src/nxdoc/nuxeo-add-ons/disaster-recovery-with-nuxeo.md

```diff
@@ -0,0 +1,448 @@
+
+This means that in a disaster recovery scenario **there will be some data loss**. That's why such scenario must be activated very carefully.
+
+In a regular application, RTO can be the time to restore the infrastructure in another DC and restore the last backup. The RPO would then be the time that has elapsed from the last backup. In that case, RPO and RTO both can be in the magnitude of hours.
+
+In our case, we want to have an RPO/RTO that is less than a single minute so the backup/restore strategy is definitively not adapted.
+
+## Prerequisites
+
+To achieve all the setup, we will use two OpenShift cluster. Those clusters should be able to communicate and they must not have the same network addressing scheme.</br>
+Here is how the setup can be done on AWS:
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Disaster Recovery with Nuxeo/Disaster Recovery Deployment
+    name: dr-deployment.png
+    addins#schema#up_to_date
+--}}
+![Disaster Recovery Deployment](nx_asset://69feb884-1c41-4654-ba27-28cb7a1b6693 ?w=450,border=true)
+
+This schema only shows the application nodes of the cluster, but the OpenShift installation should also be HA capable, meaning having three master nodes and three `etcd` nodes.
+
+## High Availability
+
+### Ha Expectations from a Nuxeo Perspective
+
+Nuxeo is a platform that relies on different middleware to persist data. Each of those middleware has to be resilient to failure:
+
+ - Metadata and directories will be stored in MongoDB. Configuring MongoDB with two replicaSet will ensure resilience to a single instance failure.
+ - Indexes will be stored in ElasticSearch. It supports HA and the shards will be replicated 2 times across data nodes.
+ - Asynchronous works and Audit will be stored in Kafka which will be deployed with 3 nodes.
+ - Binaries will be stored in an HA object store: on AWS we use S3.
+
+### Deployment in OpenShift
+
+In OpenShift the deployment is split into three parts:
+
+ - `nuxeo-backings`: takes care of the middleware deployments (MongoDB, ElasticSearch, Kafka and Zookeeper)
+ - `nuxeo-build`: takes care of the application build
+ - `nuxeo`: takes care of the Nuxeo deployment
+
+ {{!--     ### nx_asset ###
+     path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Disaster Recovery with Nuxeo/Hight Availability Deployment
+     name: ha-deployment.png
+     addins#schema#up_to_date
+ --}}
+ ![Hight Availability Deployment](nx_asset://a7609631-dd1a-4a28-bbfa-457cfe33e529 ?w=450,border=true)
+
+#### Prerequisites
+
+##### Storage Classes and Security Context Constraint
+
+The templates make use of common cluster resources which are:
+
+ - `es-scc`: a custom [Security Context Constraint](https://docs.openshift.org/3.9/admin_guide/manage_scc.html) policy that is needed by Elasticsearch deployment
```

Comment:
The problem is that this Security Context Constraint is defined in the https://github.com/nuxeo-sandbox/nuxeo-hadr private repository, like all other files referenced in that documentation.